lib/rabl/helpers.rb

```diff
@@ -81,8 +83,10 @@ def is_object?(obj, follow_symbols = true)
     # is_collection?([]) => true
     def is_collection?(obj, follow_symbols = true)
       data_obj = follow_symbols ? data_object(obj) : obj
-      data_obj && data_obj.respond_to?(:map) && data_obj.respond_to?(:each) &&
-        !(data_obj.is_a?(Struct) || defined?(Hashie::Mash) && data_obj.is_a?(Hashie::Mash))
+      data_obj &&
+        data_obj.is_a?(Enumerable) &&
```

Comment:
This seems to be a breaking change for my app when I upgraded from `0.14.0` to `0.14.1`. I'm not sure if it is related to https://github.com/nesquena/rabl/issues/724.

I have a collection that responds to map, but it is not an enumerable.

Reply:
Do ActiveRecord relations include Enumerable? It looks like they do not in rails 4.2.

```
> Widget.all.class
=> Widget::ActiveRecord_Relation
> Widget.all < Enumerable
=> nil
> Widget.all.respond_to?(:each)
=> true
> Widget.all.respond_to?(:map)
=> true
```